src/views/Register.vue

```diff
@@ -88,8 +90,12 @@ export default {
     };
   },
   methods: {
-    handleSubmit() {
-      console.log(this.model);
+    async handleSubmit() {
+      try {
```

Comment:
Add a loading state for this page. 
When user press the register button, we should display a loading state until the response from server is received

Reply:
```
async onSubmit() {
      try {
        this.loading = true
        await this.$store.dispatch('auth/login', this.model)
        await this.$router.push('/reports')
      } finally {
        this.loading = false
      }
    },
```

Reply:
something like that, and you can use the loading variable to disable the button in the form so you can't press it multiple times and call the BE multiple times